src/java.base/share/classes/java/net/InterfaceAddress.java

```diff
@@ -42,10 +42,10 @@ public class InterfaceAddress {
     private short        maskLength = 0;
 
     /*
-     * Package private constructor. Can't be built directly, instances are
-     * obtained through the NetworkInterface class.
+     * Private constructor. Not to be constructed by Java code, instances
+     * are obtained from native code through the NetworkInterface class.
```

Comment:
We should word like "this constructor is called via JNI in NetworkInterface.c" so we know what care we need to take when we update this constructor.

Reply:
> We should word like "this constructor is called via JNI in NetworkInterface.c" so we know what care we need to take when we update this constructor.

Thanks! Yes, simpler to just say where the constructor is called from. It folllows from this that instances are not meant to be created from Java code.

Pointing out that it's private adds little value as its now obvious from the access modifier.